zerver/views/streams.py

```diff
@@ -180,8 +180,18 @@ def update_stream_backend(
         do_change_stream_invite_only(stream, is_private, history_public_to_subscribers)
     return json_success()
 
-def list_subscriptions_backend(request: HttpRequest, user_profile: UserProfile) -> HttpResponse:
-    return json_success({"subscriptions": gather_subscriptions(user_profile)[0]})
+@has_request_variables
+def list_subscriptions_backend(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    include_subscribers: bool=REQ(validator=check_bool, default=False),
+) -> HttpResponse:
+    return json_success({
+        "subscriptions": gather_subscriptions(
+            user_profile,
+            include_subscribers=include_subscribers
+        )[0]
+    })
```

Comment:
This line-wrapping is a bit atypical; let's just use a local variable for the result of `gather_subscriptions`.